docs/developer-guide/running-substreams.md

```diff
@@ -77,3 +77,36 @@ map_transfers: message "eth.erc721.v1.Transfers": {
 ```
 
 The example output shown above contains data for different transfers from data in the blockchain. These transfers can also be verified on [Etherscan](https://etherscan.io/tx/0xcfb197f62ec5c7f0e71a11ec0c4a0e394a3aa41db5386e85526f86c84b3f2796).
+
+## Development & Production Mode
+
+### Development Mode
+
+Production and development modes impact the execution of Substreams. Key aspects of execution include:
+
+- The time required to reach the first byte.
+- The speed that large ranges get executed.
+- The module logs and outputs that are sent back to the client.
+
+Differences between production and development modes include:
+
+- Forward execution is enabled in production mode and disabled in development mode.
```

Comment:
Let's link `Forward execution` to where it's defined, let's also use `Forward parallel execution`

Reply:
Wording updated to add "parallel". Unsure where to link exactly. The changes will be available in the next commit/push.